fe4u-boilerplate-main/src/js/Lab2/generateSelectionOption.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable */
+
+const CourseEnum = [
+    'Mathematics',
+    'Physics',
+    'English',
+    'Computer Science',
+    'Dancing',
+    'Chess',
+    'Biology',
+    'Chemistry',
+    'Law',
+    'Art',
+    'Medicine',
+    'Statistics',
+];
+const MonthEnum = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+];
+
+function generateOption(ageRange) {
+    let selectOptionMonth = document.getElementById("fmonth")
```

Comment:
When declaring some variable, always start with `const`. You may later change it with `let`.
Here it's definately `const`, as you don't change it later in the block.